ckanext/canada/helpers.py

```diff
@@ -861,6 +861,7 @@ def ckan_to_cdts_breadcrumbs(breadcrumb_content):
         anchor = breadcrumb.find('a')
         cdts_breadcrumbs.append({
             'title': breadcrumb.text if not anchor else anchor.text,
+            'acronym': '' if not anchor else anchor.get('title'),
```

Comment:
@RabiaSajjad cannot remember, I think I had problems with CDTS and the Jinja JSON filter with None/null values. Add empty string as default value for title?

```
anchor.get('title', '')
```

Reply:
I had no issues in testing, but I'll default to empty string in case I have missed anything.